From luca.risolia@studio.unibo.it Tue Feb  7 15:49:05 2006
Date: Wed, 8 Feb 2006 00:50:59 +0000
From: Luca Risolia <luca.risolia@studio.unibo.it>
To: Greg Kroah-Hartman <gregkh@suse.de>
Subject: USB: CREDITS: Add credits about the ZC0301 and ET61X[12]51 USB drivers
Message-ID: <20060208005059.GA10459@studio.unibo.it>
Content-Disposition: inline

This patch adds credits about the ZC0301 and ET61X[12]51 USB drivers
which have been included in the mainline kernel recently.

Signed-off-by: Luca Risolia <luca.risolia@studio.unibo.it>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 CREDITS |    2 ++
 1 file changed, 2 insertions(+)

--- gregkh-2.6.orig/CREDITS
+++ gregkh-2.6/CREDITS
@@ -2814,6 +2814,8 @@ E: luca.risolia@studio.unibo.it
 P: 1024D/FCE635A4 88E8 F32F 7244 68BA 3958  5D40 99DA 5D2A FCE6 35A4
 D: V4L driver for W996[87]CF JPEG USB Dual Mode Camera Chips
 D: V4L2 driver for SN9C10x PC Camera Controllers
+D: V4L2 driver for ET61X151 and ET61X251 PC Camera Controllers
+D: V4L2 driver for ZC0301 Image Processor and Control Chip
 S: Via Liberta' 41/A
 S: Osio Sotto, 24046, Bergamo
 S: Italy
